d2/src/main/java/com/linkedin/d2/balancer/servers/ZooKeeperAnnouncer.java

```diff
@@ -185,8 +194,7 @@ public void onError(Throwable e)
       }
 
       @Override
-      public void onSuccess(None result)
-      {
+      public void onSuccess(None result) {
```

Comment:
Looks like the style in this file is to have the opening curly brace on a separate line.  It would be good to match the existing style & move your curly brace down onto the next line.

Reply:
Thanks! Done